### DISCUSSION TITLE
Anvil Private Key

### DISCUSSION THREAD

**Discussion by warrenbeufesche** [Cyfrin/foundry-full-course-cu#232] [General]
In Lesson 7 when I run in Terminal: "make anvil" I can see the Private Key for about a minute or so. Then I can no longer scroll to the top and retrieve the key. I don't see any other reports of this at all. I have Ubuntu 22 and VS Code.

**Comment by ShubhSensei** [comment]
U should run the command `anvil` only & not `make anvil` in the terminal then the private keys will stay there until you kill the terminal.

**Reply by warrenbeufesche** [reply]
Thank you so much!

**Reply by ShubhSensei** [reply]
> Thank you so much!

Can u mark my reply as 'Mark as Answer' if my answer solved your problem
